pkg/vacuum/vacuum.go

```diff
@@ -301,9 +346,35 @@ func (e *Engine) Run(ctx context.Context) {
 	}
 }
 
+func (e *Engine) vacuumChunksMissingStats(ctx context.Context, con *pgxpool.Conn) (finished bool, err error) {
+	// we look for compressed chunks that have never been vacuum and seem to be missing
+	// statistics. autovacuum will ignore these until they hit vacuum_freeze_min_age, so let's
+	// catch these early
+	// we only do a max of 3000 before we give vacuumChunksToFreeze another turn
+	for i := 0; i <= 3; i++ {
+		chunks, err := e.listChunks(ctx, con, sqlListChunksMissingStats)
+		if err != nil {
+			log.Error("msg", "failed to list compressed chunks missing stats", "error", err)
+			return false, err
+		}
+		tablesNeedingVacuum.Set(float64(len(chunks)))
+		if len(chunks) == 0 {
+			log.Info("msg", "zero compressed chunks missing stats")
```

Comment:
Should we changed this Info to Debug? Do you think it's worth it to report this as telemetry metrics?

Reply:
As configured, you'd get two of these messages every 10 minutes. Is that too frequent?

`tablesNeedingVacuum` is a gauge that exposes it.

Reply:
since we expose it in telemetry, i'll make these DEBUG